report_aeroo/static/src/js/report/reportactionmanager.js

```diff
@@ -18,12 +18,17 @@ ActionManager.include({
         if (c_action.report_type !== 'aeroo') {
             return self._super(action, options);
         }
-        
+
         framework.blockUI();
         var aeroo_url = 'report/aeroo/' + c_action.report_name;
-        if(c_action.context.active_ids){
-            aeroo_url += '/' + c_action.context.active_ids.join(',');
+        if (_.isUndefined(action.data) || _.isNull(action.data) || (_.isObject(action.data) && _.isEmpty(action.data))) {
+            if (action.context.active_ids) {
+                aeroo_url += '/' + c_action.context.active_ids.join(',');
+                // odoo does not send context if no data, but I find it quite useful to send it regardless data or no data
+                aeroo_url += '?context=' + encodeURIComponent(JSON.stringify(c_action.context));
```

Comment:
This line is the main difference with odoo. I've added for compatibility with v8/v9 where you could send values on the context to the report.
I don't see any reason why if there is not data you should not preserve the context.